content/docs/getting_started/visual_story/_blueprint.yaml

```diff
@@ -0,0 +1,5 @@
+$path: /docs/getting_started/visual_story/{base}.html
+$view: /views/doc.html
+$localization:
+  path: /{locale}/docs/getting_started/visual_story/{base}.html
+$title@: Create a visual AMP Story
```

Comment:
Can we rename this to "Create your first AMP Story" to better match the other tutorial?